Controllers/AbstractControllers/AbstractUserController.php

```diff
@@ -77,29 +103,35 @@ public function newAction(array $params = null): void
         }
 
         try {
-            $emptyResult = $this->dataValidator->emptyCheck($this->postInfo->getPostData());
+            $this->getMsgModel(self::REFERER);
+            if ($params !== null) {
+                throw new \Exception('Params have to be empty in createAction!');
```

Comment:
So you throw the exception here in this function and catch it here as well? What is the purpose of this move?